apps/api/v2/src/ee/bookings/2024-08-13/services/output.service.ts

```diff
@@ -0,0 +1,128 @@
+import { BookingsRepository_2024_08_13 } from "@/ee/bookings/2024-08-13/bookings.repository";
+import { Injectable } from "@nestjs/common";
+import { plainToClass } from "class-transformer";
+import { DateTime } from "luxon";
+import { z } from "zod";
+
+import { BookingOutput_2024_08_13, RecurringBookingOutput_2024_08_13 } from "@calcom/platform-types";
+import { Booking } from "@calcom/prisma/client";
+
+export const bookingResponsesSchema = z.object({
+  email: z.string(),
+  name: z.string(),
+  guests: z.array(z.string()).optional(),
+  rescheduledReason: z.string().optional(),
+});
+
+type DatabaseBooking = Booking & {
+  attendees: {
+    name: string;
+    email: string;
+    timeZone: string;
+    locale: string | null;
+    noShow: boolean | null;
+  }[];
+} & { user: { id: number; name: string | null; email: string } | null };
+
+@Injectable()
+export class OutputBookingsService_2024_08_13 {
+  constructor(private readonly bookingsRepository: BookingsRepository_2024_08_13) {}
+
+  getOutputBooking(databaseBooking: DatabaseBooking) {
+    const dateStart = DateTime.fromISO(databaseBooking.startTime.toISOString());
+    const dateEnd = DateTime.fromISO(databaseBooking.endTime.toISOString());
+    const duration = dateEnd.diff(dateStart, "minutes").minutes;
+
+    const bookingResponses = bookingResponsesSchema.parse(databaseBooking.responses);
+    const attendee = databaseBooking.attendees.find((attendee) => attendee.email === bookingResponses.email);
+
+    if (!attendee) {
+      throw new Error("Attendee not found");
```

Comment:
should we add the email of the attendee that was not found

Reply:
Actually this variable was unused so i removed it in [refactor: remove unused attendee variable](https://github.com/calcom/cal.com/pull/16200/commits/7c99e0f4686ff63691a29275ea26223329f3fec2)